test_cli/package.xml

```diff
@@ -6,11 +6,13 @@
   <description>
     Test command line arguments passed to ros2 executables.
   </description>
-  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
-  <maintainer email="william@openrobotics.org">William Woodall</maintainer>
+  <maintainer email="aditya.pande@osrfoundation.org">Aditya Pande</maintainer>
+  <maintainer email="shane@osrfoundation.org">Shane Loretz</maintainer>
```

Comment:
I believe we have a preference to use the "openrobotics.org" domain ([for example](https://github.com/ros2/rclpy/blob/f2cb25b215781bd673457dd4feb82c98b7ffcb70/rclpy/package.xml#L8)).

Reply:
Done !